kotlinx-collections-immutable/build.gradle

```diff
@@ -25,6 +25,7 @@ allprojects {
     dependencies {
         testCompile "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
         testCompile "junit:junit:4.12"
+        testCompile 'com.google.guava:guava-testlib:18.0'
```

Comment:
Should be added to the dependencies of `tests` project only

Reply:
+1